### ISSUE TITLE
Keg Status Which Makes a Keg Available to Tap.

### ISSUE THREAD

**Issue by EmDubeu** [RaspberryPints/RaspberryPints#138]
So, I'm a little confused regarding the Keg Status when it comes time to tap a keg. I understand only certain Keg statuses will allow kegs to show up in the tap a keg selection list. My confusion is why does the keg need a status of Primary, Secondary, Bulk Aging, Dry Hopping before tapping it? I wouldn't choose a keg that has a status of Primary or Secondary as I don't directly tap from a keg that is in primary or secondary. I understand Dry Hopping and Bulk Aging, but not the others. Also, I keep kegs for the ready with sanitizer in them so when I need to add beer to them, from my fermentor, it's ready to go. I think I would only ever tap a keg from an available keg list that had a status like "Conditioning" or "Sanitized".

**Comment by Boombrewer** [comment]
Ekim, I agree with you but I think the major thing we're missing with this one is that we're trying to make this how we each do it rather than let the user determine use like we've done with nearly everything else.  I like the idea a serving keg drops off the list of available to tap.  Besides that, the status of the keg really shouldn't have any affect in my opinion.  I use my kegs the same way Ekim does.  But I don't want the way I do things to influence others.

**Comment by thadius856** [comment]
We just hashed this one out in IRC. Here's the solution we found:

"FLOODED" is removed as a keg status. This is same thing is "Needs Cleaning", just we're indicating we haven't released the CO2, which is, for all intents and purposes, useless.

"SANITIZED" is now "CLEAN", which could mean either empty or full in the user's mind. This is good, because they won't remember to update RPints after they fill a keg.

"BULK_AGING" is now "CONDITIONING". Because reasons.

"SANITIZED" ("CLEAN") is not longer hidden from the list because we've seen people confused a couple times. Lesson learned.

Now, only "SERVING", "NEEDS CLEANING", "NEEDS PARTS" and "NEEDS REPAIRS" are hidden.
